tests/uninstall_test.go

```diff
@@ -0,0 +1,91 @@
+// +build integration
+
+package tests
+
+import (
+	"os"
+	"testing"
+
+	"get.porter.sh/porter/pkg/porter"
+	"github.com/stretchr/testify/require"
+)
+
+func TestUninstall_DeleteInstallation(t *testing.T) {
+	testcases := []struct {
+		name               string
+		installed          bool
+		uninstallFails     bool
+		delete             bool
+		forceDelete        bool
+		installationExists bool
+		wantError          string
+	}{
+		{"not yet installed", false, false, false, false, false, "1 error occurred:\n\t* could not load installation HELLO: Installation does not exist\n\n"},
```

Comment:
I suggest test cases like this, which are really long, from setting every field without using the field name (which is quite hard to read without the support of an editor) to using the field names and only setting the ones that are non-default values,

```go
{name: "not yet installed", wantError: "1 error occurred..."},
```